src/main/java/cloud/fogbow/ras/core/plugins/interoperability/azure/quota/AzureQuotaPlugin.java

```diff
@@ -0,0 +1,296 @@
+        return this.buildQuota(computeAllocation, networkAllocation, publicIpAllocation, volumeAllocation);
+    }
+
+    @VisibleForTesting
+    VolumeAllocation getTotalVolumeAllocation() {
+        int volumes = FogbowConstants.UNLIMITED_RESOURCE;
+        int storage = MAXIMUM_STORAGE_ACCOUNT_CAPACITY;
+        return new VolumeAllocation(volumes, storage);
+    }
+
+    @VisibleForTesting
+    PublicIpAllocation getTotalPublicIpAllocation(Map<String, NetworkUsage> networkUsages) {
+        NetworkUsage publicIpUsage = networkUsages.get(QUOTA_PUBLIC_IP_ADDRESSES);
+        int instances = publicIpUsage == null ? NO_USAGE : (int) publicIpUsage.limit();
+        return new PublicIpAllocation(instances);
+    }
+
+    @VisibleForTesting
+    NetworkAllocation getTotalNetworkAllocation(Map<String, NetworkUsage> networkUsages) {
+        NetworkUsage networkUsage = networkUsages.get(QUOTA_NETWORK_INSTANCES);
+        int instances = networkUsage == null ? NO_USAGE : (int) networkUsage.limit();
+        return new NetworkAllocation(instances);
+    }
+
+    @VisibleForTesting
+    ComputeAllocation getTotalComputeAllocation(Map<String, ComputeUsage> computeUsages) {
+        ComputeUsage vmUsage = computeUsages.get(QUOTA_VM_INSTANCES_KEY);
+        ComputeUsage coreUsage = computeUsages.get(QUOTA_VM_CORES_KEY);
+
+        int instances = vmUsage == null ? NO_USAGE : (int) vmUsage.limit();
+        int cores = coreUsage == null ? NO_USAGE : (int) coreUsage.limit();
+        int ram = FogbowConstants.UNLIMITED_RESOURCE;
+
+        return new ComputeAllocation(cores, ram, instances);
+    }
+
+    @VisibleForTesting
+    ResourceAllocation buildQuota(ComputeAllocation computeAllocation,
+                                  NetworkAllocation networkAllocation,
+                                  PublicIpAllocation publicIpAllocation,
+                                  VolumeAllocation volumeAllocation) {
+        int instances = computeAllocation.getInstances();
+        int ram = computeAllocation.getRam();
+        int vCPU = computeAllocation.getvCPU();
+        int volumes = volumeAllocation.getInstances();
+        int storage = volumeAllocation.getStorage();
+        int networks = networkAllocation.getInstances();
+        int publicIps = publicIpAllocation.getInstances();
+
+        return ResourceAllocation.builder()
+                .instances(instances)
+                .vCPU(vCPU)
+                .ram(ram)
+                .volumes(volumes)
+                .storage(storage)
+                .networks(networks)
+                .publicIps(publicIps)
+                .build();
+    }
+
+    @VisibleForTesting
+    Map<String, ComputeUsage> getComputeUsageMap(Azure azure) {
+        Map<String, ComputeUsage> computeUsageMap = new HashMap<>();
+        List<String> validComputeUsages = Arrays.asList(QUOTA_VM_INSTANCES_KEY, QUOTA_VM_CORES_KEY);
+
+        this.getComputeUsage(azure)
```

Comment:
You are using two indentation styles(**.stream()**) in this class, one with breakline and another without. I think Fogbow style uses without.

Reply:
Done.